conductr_cli/sandbox_start.py

```diff
@@ -0,0 +1,159 @@
+from conductr_cli import terminal
+
+
+#############################
+###       Constants       ###
+#############################
+CONDUCTR_DEV_IMAGE = 'typesafe-docker-registry-for-subscribers-only.bintray.io/conductr/conductr-dev'
+LATEST_CONDUCTR_VERSION = '1.0.11'
+CONDUCTR_NAME_PREFIX = 'cond-'
+CONDUCTR_PORTS = { 5601, # conductr-kibana bundle
+                   9004, # ConductR internal akka remoting
+                   9005, # ConductR controlServer
+                   9006, # ConductR bundleStreamServer
+                   9200, # conductr-elasticsearch bundle
+                   9999} # visualizer bundle
+
+
+#############################
+###   Public functions    ###
+#############################
+def run(args):
+    """`sandbox run` command"""
+    # TODO: pull_image(args)
+    ports = collect_ports(args)
+    scale_cluster(args, ports)
+
+def debug(args):
+    """`sandbox debug` command"""
+    print("TODO: Implement")
```

Comment:
I'm starting to wonder about the utility of debugging outside of an build environment such as sbt. WDYT?